test/test_utils/ec2.py

```diff
@@ -37,8 +37,7 @@ def get_ec2_instance_type(default, processor, disable_p3dn=False):
             "Default instance type should never be p3dn.24xlarge"
         )
     instance_type = os.getenv(f"EC2_{processor.upper()}_INSTANCE_TYPE", default)
-    #Not running eia test on p3dn instances
-    if instance_type == p3dn and (disable_p3dn or processor == "eia"):
+    if instance_type == p3dn and disable_p3dn:
```

Comment:
is this function used for eia at all? it probably can be instead of get_ec2_accelerator type

Reply:
No, This function is used for CPU and GPU processor type.
To Decouple them, we kept get_ec2_accelerator type to fetch the EI accelerator type.
I can combine both of them if needed.

Reply:
no need to combine in this PR, but probably we should remove "eia" from allowed_processors in line 28 if we are specifically decoupling eia from this function

Reply:
Sure, I will remove eia from allowed_processors in  line 28.